client/components/download-button/test/__snapshots__/index.js.snap

```diff
@@ -0,0 +1,58 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`DownloadButton renders a disabled button 1`] = `
+<div>
+  <button
+    class="components-button woocommerce-table__download-button"
+    disabled=""
```

Comment:
Should the value of this be `true`?

Reply:
I guess this is how the `Button` component imported from `@wordpress/components` behaves. Also, I think the mere presence of the `disabled` attribute implies truthiness: https://stackoverflow.com/a/32745315/1775160.

Reply:
yep, looks like we are good here!